app/src/main/java/com/pvp/app/ui/screen/calendar/TaskFormsCreate.kt

```diff
@@ -163,241 +161,210 @@ fun TaskCreateForm(
         }
     }
 
-    Surface(
```

Comment:
In whole file here and `TaskFormsEdit` - only removed redundant `Surface` wrapper, changed `when`, `if` into one `if` chain, moved `Slider` out, put `Picker` in (duration; reminderTime).

This creation form also had task type selector, so I transferred it into a different private composable to make it easier to read.